archivers/afio/Portfile

```diff
@@ -27,6 +27,9 @@ github.tarball_from archive
 patchfiles          Makefile.patch \
                     implicit.patch
 
+# cc1: error: unrecognized command line option "-Wno-unused-result"
+patchfiles-append   patch-cflags.diff
```

Comment:
so when/why did this break - the build bots have many successful attempts so it's correct/fine for many compilers/systems.

Reply:
There are no buildbots with Xcode gcc. It was broken once that flag was added or possibly always broken.